keyboards/coseyfannitutti/mysterium/mysterium.h

```diff
@@ -84,6 +84,6 @@
     { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, _x_, K1F, K1G, K1H }, \
```

Comment:
K01 above this line should be `XXX`, no?

Reply:
I'm new to QMK, I'm just following the rest of the header file format. Is xxx more common?

Reply:
Here it is defined as `_x_`, but the point is the same.

Reply:
Yes your right, I forgot to edit that for the 7u